doc/changelogs/CHANGELOG_V14.md

```diff
@@ -45,6 +45,80 @@
   * [io.js](CHANGELOG_IOJS.md)
   * [Archive](CHANGELOG_ARCHIVE.md)
 
+<a id="14.13.1"></a>
+## 2020-10-07, Version 14.13.1 (Current), @BethGriggs prepared by @danielleadams
+
+### Notable Changes
+
+* **fs**:
+  * remove experimental from rmdir recursive (Benjamin Coe) [#35171](https://github.com/nodejs/node/pull/35171)
+
+### Commits
+
+* [[`52657b3fdc`](https://github.com/nodejs/node/commit/52657b3fdc)] - Working on v14.13.1 (Myles Borins) [#35419](https://github.com/nodejs/node/pull/35419)
```

Comment:
We should drop the two release commits from here

Reply:
in future you can run branch diff with `--filter-release` to avoid those commits. You can also do `git node release --prepare` which will magically make the release commit for you. shoutout to @codebytere for that magic

Reply:
Oh nice! That's good to know